tests/SprykerEcoTest/Zed/Inxmail/InxmailTest.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEcoTest\Zed\Inxmail;
+
+use Codeception\Test\Unit;
+use Generated\Shared\Transfer\CustomerTransfer;
+use Generated\Shared\Transfer\InxmailRequestTransfer;
+use Generated\Shared\Transfer\OrderTransfer;
+use SprykerEco\Zed\Inxmail\Business\Api\Adapter\AdapterInterface;
+use SprykerEco\Zed\Inxmail\Business\Handler\Customer\CustomerEventHandler;
+use SprykerEco\Zed\Inxmail\Business\Handler\Customer\CustomerEventHandlerInterface;
+use SprykerEco\Zed\Inxmail\Business\Handler\Order\OrderEventHandler;
+use SprykerEco\Zed\Inxmail\Business\Handler\Order\OrderEventHandlerInterface;
+use SprykerEco\Zed\Inxmail\Business\InxmailBusinessFactory;
+use SprykerEco\Zed\Inxmail\Business\InxmailFacade;
+use SprykerEco\Zed\Inxmail\Business\InxmailFacadeInterface;
+use SprykerEco\Zed\Inxmail\Business\Mapper\Customer\CustomerMapperInterface;
+use SprykerEco\Zed\Inxmail\Business\Mapper\Order\OrderMapperInterface;
+use SprykerEco\Zed\Inxmail\Dependency\Facade\InxmailToSalesFacadeBridgeInterface;
+
+/**
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Inxmail
+ * @group InxmailTest
+ */
+class InxmailTest extends Unit
+{
+    /**
+     * @return void
+     */
+    public function testHandleCustomerRegisterEvent()
+    {
+        $facade = $this->prepareFacade();
+        $this->assertTrue((bool)$facade->handleCustomerRegistrationEvent($this->prepareCustomerTransfer()));
+    }
+
+    /**
+     * @return void
+     */
+    public function testHandleCustomerResetPasswordEvent()
+    {
+        $facade = $this->prepareFacade();
+        $this->assertTrue((bool)$facade->handleCustomerResetPasswordEvent($this->prepareCustomerTransfer()));
```

Comment:
Method of your facade returns string. You cast it to boolean and assert to true? It will always be true. Your tests don't make any sense.

Reply:
I think you have to rework your tests.